frame/system/src/lib.rs

```diff
@@ -201,12 +201,58 @@ pub mod pallet {
 	use crate::{self as frame_system, pallet_prelude::*, *};
 	use frame_support::pallet_prelude::*;
 
+	pub mod prelude {
+		use super::*;
+		pub mod testing {
+			type AccountId = u64;
+			use super::*;
+			use sp_runtime::traits::IdentityLookup;
+
+			/// Provides a viable default config that can be used with
+			/// [`derive_impl`](`frame_support::derive_impl`) to derive a testing pallet config
+			/// based on this one.
+			///
+			/// See `Test` in the `default-config` example pallet's `test.rs` for an example of
+			/// a downstream user of this particular `TestDefaultConfig`
+			pub struct TestDefaultConfig;
```

Comment:
Given this name, I wonder if `mod testing` is redundant?

Reply:
True, I wasn't sure if you intended for there to be more things in this mod that would justify it